src/Surface.zig

```diff
@@ -4443,6 +4443,17 @@ pub fn performBindingAction(self: *Surface, action: input.Binding.Action) !bool
             return false;
         },
 
+        .copy_title_to_clipboard => {
+            const title = self.rt_surface.getTitle() orelse return false;
+
```

Comment:
We should ignore empty titles too (length zero). I don't want to clobber the clipboard for an empty title. That was noted in the original issue.

Reply:
Fixed.